src/control/server/mgmt_pool_test.go

```diff
@@ -826,25 +826,6 @@ func TestServer_MgmtSvc_PoolDestroy(t *testing.T) {
 			},
 			expSvc: svcWithState(testPoolService, system.PoolServiceStateDestroying),
 		},
-		"force=true, evict dRPC fails due to engine error": {
```

Comment:
We should have 2 test cases that verify the destroy flow completes with either of the failures occurring.

Reply:
I tried changing this test case but could not figure out how to arrange it properly. Do you have ideas on how to structure the test cases?

Reply:
I'm having a go at this, @kccain do you want me to push a Commit to the PR or paste the diff in the comments?

Reply:
> I'm having a go at this, @kccain do you want me to push a Commit to the PR or paste the diff in the comments?

Thanks Tom. Pushing a commit would be fine, making sure to use Features: pool commit pragma for CI testing.

Reply:
Pushed with pragma to test pool features and enable MD-on-SSD coverage. The test refactor should enable coverage of different code-paths by enabling customisation of serial dRPC responses.